package.json

```diff
@@ -61,5 +95,26 @@
       "@fewlines/eslint-config/typescript",
       "@fewlines/eslint-config/react-typescript"
     ]
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "roots": [
+      "<rootDir>"
+    ],
+    "moduleFileExtensions": [
+      "js",
+      "ts",
+      "tsx",
+      "json"
+    ],
+    "testPathIgnorePatterns": [
+      "<rootDir>[/\\\\](node_modules|.next)[/\\\\]"
+    ],
+    "transformIgnorePatterns": [
+      "[/\\\\]node_modules[/\\\\].+\\.(ts|tsx)$"
+    ],
+    "transform": {
+      "^.+\\.(ts|tsx)$": "babel-jest"
+    }
```

Comment:
Starting to look like something I would prefer into its own config file 😅 
I see some similarities with what I can see there https://github.com/vercel/next.js/tree/canary/examples/with-jest
Do you think there are some things to take from them or did you go with the minimal settings?

Reply:
I used this one https://github.com/vercel/next.js/blob/canary/examples/with-typescript-eslint-jest/jest.config.js as a baseline.

Regarding 
```
"moduleFileExtensions": [
      "js",
      "ts",
      "tsx",
      "json"
    ],
```

I left `js` and `json` in case we want to test `next.config.js` (dunno if there are any interest to do so), or the `config.json` in case we wan't to use this over var env.

I think the ignore patterns can be removed, like the `roots` field.

Reply:
794d75b
Let me know if it's short enough, or if I should also get ride of the `js` and `json` item.